variants/web/template.rb

```diff
@@ -0,0 +1,35 @@
+def apply_web_variant!
+  use_source_paths [__dir__]
+
+  copy_file '.eslintignore'
+  copy_file '.eslintrc'
+  copy_file '.pronto_eslint_npm.yml'
+  copy_file '.scss-lint.yml'
+
+  copy_file '.npmrc'
+
+  apply 'Gemfile.rb'
+  apply 'app/template.rb'
+  apply 'config/template.rb'
+  apply 'package.json.rb'
+
+  remove_turbolinks
+
+  after_bundle do
+    use_source_paths [__dir__]
+
+    # Fix the default rails template that does not put trailing commas
```

Comment:
Which issues does that solve? Can't the template files be fixed instead?

Reply:
For this one, the problem is with rails by default they're not using the trailing comma. But if we not doing this, it won't pass our eslint.

You can check on their template here -> https://github.com/rails/rails/blob/v6.0.2.1/railties/lib/rails/generators/rails/app/templates/app/javascript/packs/application.js.tt

Reply:
Noted